website/docs/tutorials/07-testing/03-fuzzing/strategies.md

```diff
@@ -0,0 +1,62 @@
+# Strategies
+
+While a felt is a _lingua franca_ of Cairo runtime, its whole value space is rarely covered by
+domain space in tested code.
```

Comment:
Can you dumb it down? (lingua franca, domain)

Reply:
Done.